web-gateway/conf/messages

```diff
@@ -85,3 +85,6 @@ invalid.duration=Please enter a valid duration
 invalid.units=Please enter a valid unit
 
 createUserInstructions=Here you can create a new user who can participate in and create auctions.
+instruction.createItem=Here you can create the item which you want to auction.
+instruction.placeBid=Here you can bid a price for the item under auction.
+instruction.startAuction=Here you can start the auction of the item.
```

Comment:
I think we can work this inline instructions to help the user understand better how starting an action and bidding works.
For example, once the auction has started it's not immediately clear that a user should switch accounts to proceed using the application.

also note that before the auction has started, the item page can be used for two purposes: starting the auction or editing the items details so the inline docs should reflect that.